pkg/cmd/roachtest/gorm.go

```diff
@@ -107,12 +109,15 @@ func registerGORM(r *testRegistry) {
 		t.Status("running gorm test suite and collecting results")
 
 		// Ignore the error as there will be failing tests.
-		_ = c.RunE(
+		err = c.RunE(
 			ctx,
 			node,
 			fmt.Sprintf(`cd %s && GORMDIALECT="postgres" 
 PGUSER=root PGPORT=26257 PGSSLMODE=disable go test -v 2>&1 | %s/bin/go-junit-report > %s`, gormTestPath, goPath, resultsPath),
 		)
+		if err != nil {
+			t.L().Printf("error whilst processing tests (may be expected): %#v", err)
```

Comment:
technically "while running tests" but not a big deal

also could be helpful to add this log to other ORM tests where we swallow the error, if you are up for it